### PR TITLE
Commerce 2408

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by marco-leo** [comment]
@andrea-ale-sbarra I don't like  this commit history, please change it:

- Always commit yaml file change separately
- Always commit autogenerated files together ()
- Don't mix generated files and implementation changes.

Thanks

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
@marco-leo now commit story should be fine

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
ci:test

File: headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/AttachmentDTOConverter.java [hunk 1]
```diff
@@ -62,7 +62,7 @@ public Attachment toDTO(DTOConverterContext dtoConverterContext)
 		Company company = _companyLocalService.getCompany(
 			cpAttachmentFileEntry.getCompanyId());
 
-		String portalURL = company.getPortalURL(company.getGroupId());
+		String portalURL = company.getPortalURL(0);
```
Comment (andrea-ale-sbarra) [inline headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/AttachmentDTOConverter.java:65]: @ethanbustad I applied your suggestion in this PR

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Pull request merged**